sound/soc/sof/Makefile

```diff
@@ -2,7 +2,7 @@
 
 snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
 		control.o trace.o utils.o sof-audio.o stream-ipc.o ipc4.o \
-		topology-common.o
```

Comment:
same with this commit. Topology2.0 has nothign to do with this change. You will need to do the same thing irrespective of whether you use topolpogy1.0 or 2.0